users.json

```diff
@@ -0,0 +1,119 @@
+{
+  "students":[
+    {
+      "id":"1",
+      "name": "Sarah",
+      "matriculation_number": "544827",
+      "matriculation_date": "1.10.2013",
+      "password": "password",
+      "tracked_modules": [
+        {
+          "module_id": "1",
+          "status": "urgent",
```

Comment:
The JSON specification does not permit trailing commas. If you try to parse the file it will raise a Syntax Error.